src/Tribe/Admin/Settings.php

```diff
@@ -224,6 +227,34 @@ public function add_admin_pages() {
 		$this->maybe_add_app_shop();
 	}
 
+	/**
+	 * Redirects users from an outdated help page to the updated Help Hub page in the WordPress admin.
+	 *
+	 * Checks the `page` and `post_type` query parameters, and if they match the old help page slug.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function maybe_redirect_to_help_hub(): void {
+		$page      = tribe_get_request_var( 'page' );
+		$post_type = tribe_get_request_var( 'post_type' );
+
+		// Check if the request is for the old help page.
+		if ( Plugin::POSTTYPE === $post_type && 'tec-events-help' === $page ) {
+			$new_url = add_query_arg(
+				[
+					'post_type' => Plugin::POSTTYPE,
+					'page'      => 'tec-events-help-hub',
+				],
+				admin_url( 'edit.php' )
+			);
+
+			wp_safe_redirect( $new_url );
+			exit;
+		}
```

Comment:
Invert the logic, pull the page slug from the objects you created instead of static strings.